roles/hosted_engine_setup/tasks/create_target_vm/01_create_target_hosted_engine_vm.yml

```diff
@@ -118,7 +118,7 @@
       - {
         name: 'he_metadata',
         description: 'Hosted-Engine metadata disk',
-        size: '1GiB',
+        size: '10MiB',
```

Comment:
You will get 128 MiB disk on block storage because block storage use 128 MiB extent. So best use 128 MiB here.

Reply:
changed to 128MiB